.env.production

```diff
@@ -87,3 +87,5 @@ SPID_RELAY_STATE='appio'
 WALLET_API_BASEURL='https://api.platform.pagopa.it'
 # Wallet test/env RESTful API
 WALLET_API_UAT_BASEURL='https://api.uat.platform.pagopa.it'
+# Show new Barcode Scan section
+SHOW_BARCODE_SCAN_SECTION=NO
```

Comment:
Since we need to wait for the full implementation of the 'Services' tab (currently 'WIP by design') to enable this flag, let's add a comment before merging this PR to ensure it isn't missed when the time comes. What do you think?

Reply:
yes! Done!